merchants/verkkokauppa/order/types.py

```diff
@@ -155,12 +161,10 @@ def from_json(cls, json: Dict[str, Any]) -> "Order":
                     first_name=json["customer"]["firstName"],
                     last_name=json["customer"]["lastName"],
                     email=json["customer"]["email"],
-                    phone=json["customer"]["phone"],
+                    phone=json["customer"].get("phone", ""),
                 ),
                 status=json["status"],
-                subscription_id=UUID(json["subscriptionId"])
-                if json["subscriptionId"]
-                else None,
+                subscription_id=subscription_id if subscription_id else None,
```

Comment:
```suggestion
                subscription_id=subscription_id,
```
I have a feeling `if` statement is not needed here. On line 106 it is set to UUID and if that fails, it is set to None on line 108. The value should be safe to use use as is - unless I missed something 😄